dmriprep/utils/sentry.py

```diff
@@ -0,0 +1,208 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Stripped out routines for Sentry"""
```

Comment:
I would prefer to force running with `--notrack` for now instead of setting up sentry. We'll eventually enable it, but it seems to me that we would be more interested in sharing this code with fmriprep in a small dependency and keep both in sync.

Reply:
aahh k didn't know about the `--notrack` flag